src/cswinrt/main.cpp

```diff
@@ -347,6 +386,7 @@ Where <spec> is one or more of:
         {
             w.write(" error: %\n", e.what());
             result = 1;
+            throw e;
```

Comment:
Do we want to throw here?  The idea is print the error message to console and then return an exit code.